packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -553,34 +553,37 @@ class CupertinoPageTransitionsBuilder extends PageTransitionsBuilder {
 ///
 ///  * [ThemeData.pageTransitionsTheme], which defines the default page
 ///    transitions for the overall theme.
-///  * [FadeUpwardsPageTransitionsBuilder], which defines a default page transition.
+///  * [FadeUpwardsPageTransitionsBuilder], which defines a page transition
+///    that's similar to the one provided by Android O.
 ///  * [OpenUpwardsPageTransitionsBuilder], which defines a page transition
 ///    that's similar to the one provided by Android P.
+///  * [ZoomPageTransitionsBuilder], which defines the default page transition
+///    that's similar to the one provided by Android Q.
 ///  * [CupertinoPageTransitionsBuilder], which defines a horizontal page
 ///    transition that matches native iOS page transitions.
 @immutable
 class PageTransitionsTheme with Diagnosticable {
   /// Constructs an object that selects a transition based on the platform.
   ///
-  /// By default the list of builders is: [FadeUpwardsPageTransitionsBuilder]
+  /// By default the list of builders is: [ZoomPageTransitionsBuilder]
   /// for [TargetPlatform.android], and [CupertinoPageTransitionsBuilder] for
   /// [TargetPlatform.iOS] and [TargetPlatform.macOS].
   const PageTransitionsTheme({ Map<TargetPlatform, PageTransitionsBuilder> builders = _defaultBuilders }) : _builders = builders;
 
   static const Map<TargetPlatform, PageTransitionsBuilder> _defaultBuilders = <TargetPlatform, PageTransitionsBuilder>{
-    TargetPlatform.android: FadeUpwardsPageTransitionsBuilder(),
+    TargetPlatform.android: ZoomPageTransitionsBuilder(),
     TargetPlatform.iOS: CupertinoPageTransitionsBuilder(),
-    TargetPlatform.linux: FadeUpwardsPageTransitionsBuilder(),
+    TargetPlatform.linux: ZoomPageTransitionsBuilder(),
     TargetPlatform.macOS: CupertinoPageTransitionsBuilder(),
-    TargetPlatform.windows: FadeUpwardsPageTransitionsBuilder(),
+    TargetPlatform.windows: ZoomPageTransitionsBuilder(),
```

Comment:
It's strange to me that we use an android-specific animation on desktop for Linux and Windows. I don't think I've ever seen a desktop app before that had UX that felt android-y to me. Are we sure this is what users will want?

Reply:
Also, since we're defaulting to `ZoomPageTransitionBuilder` below, I'd advocate for removing `linux` and `windows` from the list here. It would make sense to me that we define the Cupertino animation for macOS and iOS, and the android animation for android, but I think it would make sense that all other platforms would have their defaults controlled just by modifying the default set on L602

Reply:
>  I'd advocate for removing linux and windows from the list here.

I do advocate the same opinion as yours, but here I followed the minimum changes policy as I was used to.

Reply:
Flutter does not have a minimum changes policy -- very much the opposite, we have an "embrace the yak shave" policy. :-) See the style guide.

Reply:
> Flutter does not have a minimum changes policy -- very much the opposite, we have an "embrace the yak shave" policy. :-) See the style guide.

So let's try to remove them. 😉